### PR TITLE
APID-543 - Update to point anchors to actual inputs for errors

### PR DESCRIPTION
**Pull request by Tobias1087**

### CODE REVIEW

File: run_local_with_dependencies.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-sm --start MONGO DATASTREAM EMAIL HMRC_EMAIL_RENDERER MAILGUN_STUB DESKPRO_STUB HMRCDESKPRO
+sm2 --start MONGO DATASTREAM EMAIL HMRC_EMAIL_RENDERER MAILGUN_STUB DESKPRO_STUB HMRCDESKPRO
```
Comment (AndySpaven) [inline run_local_with_dependencies.sh:3]: Not everyone is on sm2 so do not change this.  Alias it locally if you want to.

Comment (Tobias1087) [reply run_local_with_dependencies.sh:3]: Thought i'd excluded this change.

File: app/uk/gov/hmrc/apiplatform/modules/uplift/views/SellResellOrDistributeSoftwareView.scala.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1 class="govuk-fieldset__heading">
             <div class="govuk-radios govuk-radios--inline">
                 @helper.CSRF.formField
                 <div class="govuk-radios__item">
-                    <input class="govuk-radios__input" id="distribute-question-yes" name="answer" type="radio" value="Yes" @if(form.data.get("answer") == Some("Yes")) {checked}>
+                    <input class="govuk-radios__input" id="answer" name="answer" type="radio" value="Yes" @if(form.data.get("answer") == Some("Yes")) {checked}>
                     <label class="govuk-label govuk-radios__label" for="distribute-question-yes">
```
Comment (johnsgp) [inline app/uk/gov/hmrc/apiplatform/modules/uplift/views/SellResellOrDistributeSoftwareView.scala.html:51]: This label no longer matches the input's ID

File: app/views/DeletePrincipalApplicationConfirmView.scala.html [hunk 3]
```diff
@@ -40,15 +40,15 @@ <h1 class="govuk-heading-l">Delete application</h1>
...
         <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
           <h2 class="govuk-heading-m">Are you sure you want us to delete this application?</h2>
         </legend>
         @fieldError(form.errors, "deleteConfirm")
         <div class="govuk-radios govuk-radios--inline" data-module="govuk-radios">
           @helper.CSRF.formField
           <div class="govuk-radios__item">
-            <input id="yes" class="govuk-radios__input" type="radio" name="deleteConfirm" value="Yes">
+            <input id="deleteConfirm" class="govuk-radios__input" type="radio" name="deleteConfirm" value="Yes">
             <label class="govuk-label govuk-radios__label" for="yes">Yes</label>
```
Comment (johnsgp) [inline app/views/DeletePrincipalApplicationConfirmView.scala.html:52]: This label no longer matches the input above it

**Pull request merged**